mandatory/2-function-creation.js

```diff
@@ -34,7 +47,14 @@ Write a function that:
 - numbers greater 100 must be replaced with 100
 */
 
-function formatPercentage(arr) {}
+function formatPercentage(arr) {
+  for (let i = 0; i < arr.length; i++) {
+    if (arr[i] > 100) arr[i] = 100;
+    arr[i] = +arr[i].toFixed(2);
```

Comment:
I had to go look up what the unary operator did it because I wasn't familiar with it 😅 .

I'd say that you'd see `parseFloat()` used more commonly, but it's fine for you to use the unary operator instead (as long as you understand what it's doing).